modules/SaveLoad.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 require './config'
-module Save_Load
+module SaveLoad
```

Comment:
[about file and class namings](https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#project-structure)